### PR TITLE
Generate new rails app for final project

### PR DESCRIPTION
**Pull request by satorinic**
## Initial basic scaffolding of new rails app ##

PT: https://www.pivotaltracker.com/story/show/150868682

### CODE REVIEW

File: .idea/.generators [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (j0sv) [inline .idea/.generators]: .idea/* should be in .gitignore

**Review by j0sv — CHANGES_REQUESTED** [review]

File: .idea/.generators (hunk 1 above)
Comment (satorinic) [reply .idea/.generators]: done

**Review by tochman — CHANGES_REQUESTED** [review]
I would ask you to provide good PR descriptions with links to Pivotal story.

**Comment by caressahh** [comment]
GREAT STUFF!!

**Pull request merged**